components/Formular/FormularAddRecipe.js

```diff
@@ -27,35 +32,51 @@ export default function RecipeForm({ onSubmit, formName, defaultData }) {
     setIngredients(updatedIngredients);
     ingredient.focus();
   }
-
+  function handleToggleFavorite(newStatus) {
+    setIsFavorite(newStatus);
+  }
   function handleSubmit(event) {
     event.preventDefault();
     const formData = new FormData(event.target);
     const data = Object.fromEntries(formData);
-    onSubmit({ ...data, ingredients });
+    onSubmit({ ...data, ingredients, isFavorite });
   }
 
   return (
     <StyledDiv>
       <StyledForm
-        aria-labelledby={formName}
+        aria-label="Recipe Form"
         id="recipeForm"
         onSubmit={handleSubmit}
+        defaultData={{
+          isFavorite: isFavorite,
+        }}
       >
+        <FavoriteButton
+          isFavorite={isFavorite}
+          onToggleFavorite={handleToggleFavorite}
+        />
         <StyledInput>
           <label htmlFor="title">Title: </label>
           <input
             id="title"
             name="title"
             type="text"
             defaultValue={defaultData?.title}
+            aria-labelledby="title"
           />
         </StyledInput>
-
         <StyledInput>
-          <label htmlFor="preparation"> Preparation:{" "}
-            <select name="preparation" id="preparation" defaultValue={defaultData?.preparation}>
-            <option value="none">none</option>
+          <label htmlFor="preparation">
```

Comment:
The following select is embedded in this label, so you don´t need to explicitly set an `htmlFor` here.

Reply:
Hi Thomas! Thank you for your comments! To be honest, we don´t understand everything ;-) So we be happy if you could help us a bit... regarding the aria-labels: we thought it would be helpful for accessibility?  and in this case, we do not need the hmtl for because select and option is embedded in label, but 56-58 should not be removed, right?